_data/standing_meetings.yml

```diff
@@ -9,6 +9,10 @@
       date: 2022-02-16 14:00:00 +0200
       speaker: "Vassil Vassilev"
       link: "[Slides](/assets/presentations/CaaS_Weekly_16_02_2020-V_Vassilev-Q1_Q2_Project_Plans.pdf)"
+    - title: "Utilizing Clad Derivatives in RooFit - Preliminary Project Roadplan"
```

Comment:
roadmap is the more usual word. You might update that as convenient..
@grimmmyshini

Reply:
Ah I knew something was wrong with that word XD. I'll put in a quick PR maybe? (Also sorry this completely slipped my mind).